### ISSUE TITLE
About training time

### ISSUE THREAD

**Issue by tb2-sy** [fudan-zvg/4d-gaussian-splatting#7]
Thanks for your nice work! Compared with the concurrent 4D Gaussian paper, your results are better and the rendering speed is faster. I am more concerned about the training time, but I did not find this information in the paper.

**Comment by Alexander0Yang** [comment]
The training time is highly influenced by the chosen hyperparameters such as the number of the initial points, schedule of densification/pruning, batch size, the chosen of regularization, and etc.

Loss computation as well as the dataloader contributes a large portion of the training time, which can be further optimized with the improvement of the training framework. 
For example, training 30K iters on the [hellwarrior](https://github.com/fudan-zvg/4d-gaussian-splatting/blob/main/configs/dnerf/hellwarrior.yaml) took 18min on a RTX 4090 Laptop GPU. 

Actually, considering that the computation of SSIM loss constitutes the vast majority of time in each training iteration, the time cost per training iteration for the recent GS-based approaches should exhibit minimal differences. In contrast, the rendering speed is more representative of the efficiency of the proposed scene representation itself.

**Comment by tb2-sy** [comment]
> The training time is highly influenced by the chosen hyperparameters such as the number of the initial points, schedule of densification/pruning, batch size, the chosen of regularization, and etc.
> 
> Loss computation as well as the dataloader contributes a large portion of the training time, which can be further optimized with the improvement of the training framework. For example, training 30K iters on the [hellwarrior](https://github.com/fudan-zvg/4d-gaussian-splatting/blob/main/configs/dnerf/hellwarrior.yaml) took 18min on a RTX 4090 Laptop GPU.
> 
> Actually, considering that the computation of SSIM loss constitutes the vast majority of time in each training iteration, the time cost per training iteration for the recent GS-based approaches should exhibit minimal differences. In contrast, the rendering speed is more representative of the efficiency of the proposed scene representation itself.

i see, thanks for your reply!
